From def2027201d0ef17f58151010f4ec51a0532016d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=9D=8E=E5=A8=9F?= <juan.li@spacemit.com>
Date: Wed, 7 Aug 2024 10:00:28 +0800
Subject: [PATCH 1026/1448] this is not pcie patch Revert "pcie: clean debug
 info"

This reverts commit e3d7f408f15e67f4930408ffa3bc0714679c560b.

Reason for revert: <INSERT REASONING HERE>

Change-Id: Ic57a6ceb17c32b0881b8522d3ea6dc4531e53a8b
---
 drivers/tty/serial/pxa_k1x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/pxa_k1x.c b/drivers/tty/serial/pxa_k1x.c
index a00ef2129710..73be034c3da9 100644
--- a/drivers/tty/serial/pxa_k1x.c
+++ b/drivers/tty/serial/pxa_k1x.c
@@ -2075,7 +2075,7 @@ static int serial_pxa_probe_dt(struct platform_device *pdev, struct uart_pxa_por
 
 #ifdef CONFIG_PM
 	if (of_property_read_u32(np, "edge-wakeup-pin", &sport->edge_wakeup_gpio)) {
-		dev_dbg(&pdev->dev, "no edge-wakeup-pin defined.\n");
+		dev_dbg(&pdev->dev, "no edge-wakeup-pin defined\n");
 	}
 #endif
 	sport->device_ctrl_rts = of_property_read_bool(np, "device-control-rts");
-- 
2.47.0

